packages/cli/scripts/README.md

```
@@ -1,193 +1,117 @@
 
-# Local script usage: install specific version
-.\install-windows.ps1 -Version 1.2.3
+## Extension Mapping (User Agent → Download)
 
-# Skip native host configuration
-.\install-windows.ps1 -SkipNativeHost
+Suggested mapping for detecting the right installer from a browser extension:
 
-# Uninstall
-.\install-windows.ps1 -Uninstall
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)